rclcpp/include/rclcpp/rclcpp.hpp

```diff
@@ -150,8 +150,16 @@ namespace rclcpp
 // For example, this next line escalates type "rclcpp:node::Node" to "rclcpp::Node"
 using rclcpp::node::Node;
 using rclcpp::publisher::Publisher;
-using rclcpp::subscription::SubscriptionBase;
+using rclcpp::publisher::PublisherBase;
 using rclcpp::subscription::Subscription;
+using rclcpp::subscription::SubscriptionBase;
+using rclcpp::client::Client;
```

Comment:
Nit: looks like these were listed alphabetically originally

Reply:
not really (e.g. subscription before rate), so I chose to continue "grouping" them

Reply:
works for me :+1: